robot-core/src/main/java/org/obolibrary/robot/QueryOperation.java

```diff
@@ -254,6 +254,25 @@ public static OWLOntology convertModel(Model model, IOHelper ioHelper, String ca
     return ioHelper.loadOntology(new ByteArrayInputStream(os.toByteArray()), catalogPath);
   }
 
+  /**
+   * Given a Model, an IOHelper, and a path to an XML catalog, convert the model to an OWLOntology
+   * object.
+   *
+   * @param model Model to convert to OWLOntology
+   * @param ioHelper IOHelper to load ontology
+   * @param catalogPath String path to XML catalog
+   * @return OWLOntology object version of model
+   * @throws IOException on issue loading ontology
+   */
+  public static OWLOntology convertModelThroughTemporaryFile(Model model, IOHelper ioHelper, String catalogPath) throws IOException {
```

Comment:
Instead of `convertModelThroughTemporaryFile`, could we overload like so:
```java
public static OWLOntology convertModel(Model model, IOHelper ioHelper, String catalogPath) {
    return convertModel(model, ioHelper, catalogPath, false);
}

public static OWLOntology convertModel(Model model, IOHelper ioHelper, String catalogPath, boolean useTemporaryFile) {
    if (useTemporaryFile) {
        ...
    } else {
        ...
    }
}
```
I think it better matches previous patterns, and we can just call it like `convertModel(model, ioHelper, catalogPath, useTemporaryFile)` from `QueryCommand.java`.

Reply:
OK, updated.

Reply:
@beckyjackson 🤦 sorry ...

Reply:
No problem! Thanks for making this change. 😄